test/unit/services/instanceAdapter_spec.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+
+var InstanceAdapter = require("../../../lib/services/instanceAdapter");
+var _               = require("lodash");
+var expect          = require("chai").expect;
+var Genesis         = require("genesis");
+var Bluebird        = require("bluebird");
+var Sinon           = require("sinon");
+require("sinon-as-promised")(Bluebird);
+
+describe("The Adapter class ", function () {
```

Comment:
It would be worth being explicit here since there may very well be multiple adapters at some point. As it is written currently, the test output will read like "The Adapter class is frozen". It may be more helpful for it to say something like "An InstanceAdapter is immutable".